package.json

```diff
@@ -14,15 +14,16 @@
     "url": "git+https://github.com/TruSet/commit-reveal-voting.git"
   },
   "author": "Greg Taschuk <greg.taschuk@consensys.net> (https://github.com/gtaschuk) & Neil McLaren <neil.mclaren@consensys.net> (https://github.com/nmclrn)",
-  "license": "ALv2",
```

Comment:
please leave as is, this was jimjag's preferred license name

Reply:
@gtaschuk With the greatest respect to Jim, I think he got this one wrong. The string "AVLv2" is not on the list of licence names recognised by npm (see https://docs.npmjs.com/files/package.json#license and https://spdx.org/licenses/), which might be why https://www.npmjs.com/package/commit-reveal-voting still shows us using the MIT licence.

Out of interest, why did we use Apache for this repo but MIT for others? Should we change them all?

Reply:
> might be why https://www.npmjs.com/package/commit-reveal-voting still shows us using the MIT licence

It was. npm correctly shows Apache now that I published with the new string.

Reply:
We had opensourced bitmask-rbac before we spoke with Jim.